src/utilities/visibility.scss

```diff
@@ -0,0 +1,18 @@
+// --- Visibility ---
+
+.visibility-hidden {
+  visibility: hidden;
+}
+
+.visibility-visible {
+  visibility: visible;
+}
+
+.visibility-sr-only {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  margin: 0;
+  overflow: hidden;
+  visibility: hidden;
```

Comment:
It seems  `visibilty: hidden` also removes the element from the accessibility tree so it won't be picked up by assistive technology.

Reply:
@ethanWallace updated!